client/my-sites/plans/current-plan/purchases-listing.jsx

```diff
@@ -202,6 +206,49 @@ class PurchasesListing extends Component {
 		);
 	}
 
+	getProductActionButtons( purchase ) {
+		const { translate, selectedSiteSlug: site } = this.props;
+		const actionButton = this.getActionButton( purchase );
+
+		let serviceButton = null;
+		if ( isJetpackBackup( purchase ) ) {
+			serviceButton = (
+				<Button
+					href={ addQueryArgs(
+						{ site, source: 'jetpack-backup-dash' },
```

Comment:
I am not sure what this needs to be.

Reply:
Talked with folks and I resolved in 3f770fa.